OpenSim/Simulation/InverseKinematicsSolver.cpp

```diff
@@ -218,18 +218,13 @@ void InverseKinematicsSolver::setupGoals(SimTK::State &s)
     // get markers defined by the model 
     const MarkerSet &modelMarkerSet = getModel().getMarkerSet();
 
-    // get markers with specified tasks/weights
-    const Set<MarkerWeight>& mwSet = _markersReference.updMarkerWeightSet();
-    
     int index = -1;
-    int wIndex = -1;
     SimTK::Transform X_BF;
     //Loop through all markers in the reference
     for(unsigned int i=0; i < markerNames.size(); ++i){
```

Comment:
Actually, I don't think `unsigned i=0;` will cause a warning on Linux. I think the warning I was remembering would occur if it were `for(int i=0; i<markerNames.size(); ++i)` because then `i` (an int) would be compared to `markerNames.size()` (an unsigned). I prefer to be very explicit now (e.g., `unsigned i=0u;`, but it is not necessary.